docs/configuration/big_queries.md

```diff
@@ -0,0 +1,5 @@
+# Reading Large Queries
+
+When working with large queries (Queries that are larger than 2^31 Bytes or ~2GB), 
```

Comment:
```suggestion
# Huge Query Strings

When working with large query strings (longer than 2³² Bytes or ~2GB), 
```

Reply:
I moved the section to the file `queries.md`.